### ISSUE TITLE
Cannot import dmrgscf after installation:  OSError: no file with expected extension

### ISSUE THREAD

**Issue by ruixingw** [pyscf/dmrgscf#1]
Hello. I am trying to install ```pyscf``` along with ```dmrgscf```. 
For the OS, I tried both CentOS 8.3 and 7.9. I use ```conda``` to create a pure python environment, and then use ```pip``` to install both ```pyscf``` and ```dmrgscf```. The installation was successful, but cannot import ```dmrgscf```. 

I attached below how to reproduce the problem. Could you please have a look on it?

```
~> conda create -n block-pyscf python=3.8
 (successful)
~> conda activate block-pyscf
 (successful)
~> pip install pyscf
 (successful)
~> pip install git+https://github.com/pyscf/dmrgscf
 (successful)
~> pip list
 Package       Version
------------- -------------------
certifi       2020.12.5
h5py          3.2.1
numpy         1.20.3
pip           21.1.1
pyscf         1.7.6.post1
pyscf-dmrgscf 0.1.0
scipy         1.6.3
setuptools    52.0.0.post20210125
wheel         0.36.2
~> conda list
# packages in environment at /opt/toolkits/miniconda3/envs/block-pyscf:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
ca-certificates           2021.4.13            h06a4308_1    defaults
certifi                   2020.12.5        py38h06a4308_0    defaults
h5py                      3.2.1                    pypi_0    pypi
ld_impl_linux-64          2.33.1               h53a641e_7    defaults
libffi                    3.3                  he6710b0_2    defaults
libgcc-ng                 9.1.0                hdf63c60_0    defaults
libstdcxx-ng              9.1.0                hdf63c60_0    defaults
ncurses                   6.2                  he6710b0_1    defaults
numpy                     1.20.3                   pypi_0    pypi
openssl                   1.1.1k               h27cfd23_0    defaults
pip                       21.1.1           py38h06a4308_0    defaults
pyscf                     1.7.6.post1              pypi_0    pypi
pyscf-dmrgscf             0.1.0                    pypi_0    pypi
python                    3.8.10               hdb3f193_7    defaults
readline                  8.1                  h27cfd23_0    defaults
scipy                     1.6.3                    pypi_0    pypi
setuptools                52.0.0           py38h06a4308_0    defaults
sqlite                    3.35.4               hdfb4753_0    defaults
tk                        8.6.10               hbc83047_0    defaults
wheel                     0.36.2             pyhd3eb1b0_0    defaults
xz                        5.2.5                h7b6447c_0    defaults
zlib                      1.2.11               h7b6447c_3    defaults

~> python
Python 3.8.10 (default, May 19 2021, 18:05:58)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyscf import dmrgscf
/opt/toolkits/miniconda3/envs/block-pyscf/lib/python3.8/site-packages/pyscf/lib/misc.py:46: H5pyDeprecationWarning: Using default_file_mode other than 'r' is deprecated. Pass the mode to h5py.File() instead.
  h5py.get_config().default_file_mode = 'a'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/toolkits/miniconda3/envs/block-pyscf/lib/python3.8/site-packages/pyscf/dmrgscf/__init__.py", line 106, in <module>
    from pyscf.dmrgscf import dmrgci
  File "/opt/toolkits/miniconda3/envs/block-pyscf/lib/python3.8/site-packages/pyscf/dmrgscf/dmrgci.py", line 41, in <module>
    libunpack = lib.load_library('libicmpspt')
  File "/opt/toolkits/miniconda3/envs/block-pyscf/lib/python3.8/site-packages/pyscf/lib/misc.py", line 68, in load_library
    return numpy.ctypeslib.load_library(libname, _loaderpath)
  File "/opt/toolkits/miniconda3/envs/block-pyscf/lib/python3.8/site-packages/numpy/ctypeslib.py", line 153, in load_library
    raise OSError("no file with expected extension")
OSError: no file with expected extension
>>>
 ```

**Comment by ruixingw** [comment]
by the way, the current pyscf documentation is for pyscf 2.0, but on PyPI it is still 1.7.6, which introduced some confusion. (```pip install pyscf[all]``` does not work).

**Comment by WenkaiChen** [comment]
> by the way, the current pyscf documentation is for pyscf 2.0, but on PyPI it is still 1.7.6, which introduced some confusion. (`pip install pyscf[all]` does not work).

Hi, I have met same issue, have you ever solved it? (for "OSError: no file with expected extension")

**Comment by ruixingw** [comment]
> > by the way, the current pyscf documentation is for pyscf 2.0, but on PyPI it is still 1.7.6, which introduced some confusion. (`pip install pyscf[all]` does not work).
> 
> Hi, I have met same issue, have you ever solved it? (for "OSError: no file with expected extension")

No. I guess we have to wait for the developer's reply.

**Comment by gkc1000** [comment]
@fishjojo @hczhai please look into this?

The above should response, but you may get a quicker response posting to PySCF issues ... not everyone is subscribed to this list.

**Comment by jjgoings** [comment]
Any update on this? I am having the same issue with fresh installs both on Mac and Linux
```
pip install git+https://github.com/pyscf/pyscf
```
followed by
```
pip install git+https://github.com/pyscf/dmrgscf
```
yields same error as OP

**Comment by fishjojo** [comment]
It has been fixed in 22a80b9d490c10aa96b39867d5dad299600e32ae.
